scheduler/src/cook/compute_cluster.clj

```diff
@@ -93,7 +93,10 @@
   (retrieve-sandbox-url-path [this instance-entity]
     "Constructs a URL to query the sandbox directory of the task.
      Users will need to add the file path & offset to their query.
-     Refer to the 'Using the output_url' section in docs/scheduler-rest-api.adoc for further details."))
+     Refer to the 'Using the output_url' section in docs/scheduler-rest-api.adoc for further details.")
+
+  (get-compute-cluster-launch-rate-limiter [this]
```

Comment:
We haven't been using `get-` for getters, e.g. `compute-cluster-name`, `db-id`

Reply:
Renamed